.github/workflows/verify.yml

```
@@ -51,7 +45,7 @@ jobs:
 
   notification:
     if: always()
-    name: notification
+    name: Send Workflow Notification
```

```suggestion
    name: Send workflow notification
```

Following the rest of their conventions.